src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -953,4 +953,7 @@
   <data name="PowerLauncher_EnablePluginToggle.AutomationProperties.Name" xml:space="preserve">
     <value>Enable plugin</value>
   </data>
+  <data name="PowerLauncher_NotAccessibleWarning.Text" xml:space="preserve">
+    <value>The plugin is not accessible from PowerToys Run</value>

```

Comment:
Can we change the text to "This plugin can't be used. Please define an activation phrase or allow it for the global results."?

I think the suggested text is more informativ for the user.

Reply:
Agreed.
Also the terminology `The plugin is not accessible` may not be optimal to explain what the problem is.
Probably putting just `Please define an activation phrase or allow it for the global results.` is good enough.

Reply:
In my first idea I had this text: "This plugin can't be used. Please define an activation phrase or allow it for the global results."

Reply:
@htcfreek 
it's a warning, not an error, so starting the sentence with "This plugin can't be used" is not ideal.

Reply:
> @htcfreek 
> it's a warning, not an error, so starting the sentence with "This plugin can't be used" is not ideal.

@enricogior
Oh. okay. 

Then this can be good "Please define an activation phrase or allow it for the global results to use this plugin."

Or we go with your suggestion and only write this text: "Please define an activation phrase or allow it for the global results."

Reply:
@enricogior 
I personally like "Please define an activation phrase or allow it for the global results to use this plugin.". As it gives information on why a user should define an activation phrase or allow it for the global results. 
Any objections here?

Reply:
> @enricogior 
> I personally like "Please define an activation phrase or allow it for the global results to use this plugin.". As it gives information on why a user should define an activation phrase or allow it for the global results. 
> Any objections here?
> 

Regarding the wording maybe it makes sense to switch "this plugin" and 'it". We shouldn't reference to a subject before naming it. 
=> "Please define an activation phrase or allow this plugin for the global results to use it."